themes/ddbasic/sass/components/opening-hours.scss

```diff
@@ -0,0 +1,114 @@
+@import '../base.scss';
```

Comment:
Describe component contents/purpose, why are these styles grouped? What is the common denominator?